.travis.yml

```diff
@@ -20,6 +23,12 @@ before_install:
   - export PATH=/home/travis/miniconda3/bin:$PATH
   # Update conda itself
   - conda update --yes conda
+  # test that npm exists with language: node_js specified.
+  # node-qunit-puppeteer is a runner.js like interface for
```

Comment:
I do not understand the comment "... exists with language: node_js specified" ... I don't see node_js specified ... could you clarify?

Reply:
No problem. I realize now this is a comment for me; I wasn't sure if 'npm' would be available by adding 'language: node_js' up top or not. It's no longer relevant, so I'll remove it.